src/utils/components/DiskModal/DiskModal.tsx

```diff
@@ -71,7 +72,8 @@ const DiskModal: FC<DiskModalProps> = ({
         }}
         onSubmit={() =>
           handleSubmit((data) => {
-            if (isEditingCreatedDisk) return editVMDisk(vm, initialFormData, data, onSubmit);
+            if (isEditingCreatedDisk || customize)
+              return editVMDisk(vm, initialFormData, data, onSubmit);
```

Comment:
Please select a different name to this function as it's too similar to editDisk

Reply:
Fixed